component-models.json

```diff
@@ -1287,27 +1335,100 @@
       {
         "component": "aem-content",
         "valueType": "string",
-        "name": "link",
+        "name": "card_link",
```

Comment:
as discussed we should have a `card_linkText` as well to use a title for the link of the card.